.github/workflows/acceptance.yml

```diff
@@ -176,8 +119,11 @@ jobs:
             python setup.py install_egg_info
             pip install wheel # GitHub Actions does not have this installed by default (unlike Travis)
             pip install -U -e ".[dev]"
-            psql -c 'create database sentry;' -h localhost -U postgres
+
+        - name: Start devservices
+          run: |
             sentry init
+            sentry devservices up
```

Comment:
Relay isn't needed because acceptance tests don't involve event ingestion, I think. You could try `sentry devservices up postgres redis clickhouse snuba`, it'll be slightly faster.

Reply:
Updated